code/datums/components/tackle.dm

```diff
@@ -74,10 +74,7 @@
 	if(modifiers[ALT_CLICK] || modifiers[SHIFT_CLICK] || modifiers[CTRL_CLICK] || modifiers[MIDDLE_CLICK])
 		return
 
-	if(!modifiers[RIGHT_CLICK])
```

Comment:
The right-click check was intentional here to prevent accidental tackles.
It was added in #84909